lib/collection/property.js

```diff
@@ -52,8 +53,9 @@ _.inherit((
              * @note The property can also be present in the `postman_id` meta in case it is not specified in the
              * object. An auto-generated property is used wherever one is not specified
              */
-            id: src.id || (this._ ? this._['postman_id'] : undefined),
-            /*jshint +W069 */
+            id: src.id || (this._ ? this._postman_id : undefined),
```

Comment:
Good find. Seems that this is a bug introduced. It appears that the reference branch commit has been directly merged instead of being used as a reference.